api-handler/dataAccess.js

```diff
@@ -5,12 +5,16 @@ const cul = require("./getAllCultures");
 const hou = require("./getAllHouses");
 const reg = require("./getAllRegions");
 const tit = require("./getAllTitles");
+const pop = require("./getPopularity");
+const rel = require("./support/deadRelations");
 
 const url_cha = 'https://got-api.bruck.me/api/characters/';
```

Comment:
Still :D I don't like seeing bruck's server hardcoded anywhere, no matter if or not used :D Actually, if it's not used, why is it even there?